src/TrezorKeyring.test.ts

```diff
@@ -38,10 +44,13 @@ const fakeAccounts = [
   '0xd4F1686961642340a80334b5171d85Bbd390c691',
   '0x6772C4B1E841b295960Bb4662dceD9bb71726357',
   '0x41bEAD6585eCA6c79B553Ca136f0DFA78A006899',
-];
+] as const;
 
 const fakeXPubKey =
   'xpub6FnCn6nSzZAw5Tw7cgR9bi15UV96gLZhjDstkXXxvCLsUXBGXPdSnLFbdpq8p9HmGsApME5hQTZ3emM2rnY5agb9rXpVGyy3bdW6EEgAtqt';
+// `fromExtendedKey` is missing from `@types/hdkey`
```

Comment:
Perhaps we could fix this by updating `@types/hdkey` to v2.0.1 instead? That's the next available version after v0.7.0, which is what we're using. It does seem to include this method.

Not ideal to use a types package that is a different version than the package, but they already don't match. The breaking changes for v1 and v2 seem to be related to dropping support for older Node.js versions, not for any compatibility change, so it might be close enough for our purposes.